test/module/irohad/ametsuchi/postgres_executor_test.cpp

```diff
@@ -64,43 +64,51 @@ namespace iroha {
         AmetsuchiTest::TearDown();
       }
 
-      CommandResult execute(
-          const std::unique_ptr<shared_model::interface::Command> &command,
-          bool do_validation = false,
-          const shared_model::interface::types::AccountIdType &creator =
-              "id@domain") {
-        executor->doValidation(not do_validation);
-        executor->setCreatorAccountId(creator);
-        return boost::apply_visitor(*executor, command->get());
+      /**
+       * Check that passed result contains value and not an error (without this
+       * method the one below is not able to perform this check
+       * @param result to be checked
+       */
+      void checkCommandResult(const CommandResult &result) {
+        ASSERT_TRUE(val(result));
```

Comment:
Now it's impossible to understand which assert has failed

Reply:
Maybe move ASSERT_TRUE to top level, and execute to return val(result)

Reply:
We cannot return val(result), because it is not going to work for error cases, where we cannot unwrap result to get value